examples/step-67/step-67.cc

```diff
@@ -15,10 +15,14 @@
 
  *
  * Author: Martin Kronbichler, 2020
+ * Author: Stefano Zampini, King Abdullah University of Science and Technology,
+ 2022
```

Comment:
It would have been nice if we had talked before you make yourself a co-author of a tutorial program written by me. Note that several other authors have implicitly contributed to this tutorial program, see here: https://github.com/dealii/dealii/blob/0ba3b2b17df87e6aa3ac3a1a7efb5ae8a2fe2392/examples/step-67/doc/intro.dox#L5-L7 Thus, I feel uncomfortable to make this distinction like you suggest in terms of the added lines, in particular since the time stepping is not the primary intent of the tutorial.

Reply:
Apologies, it was not my intent to steal your work.